recipes/computer_vision/object_detection/app/object_detection_client.py

```diff
@@ -6,7 +6,7 @@
 import io
 
 st.title("🕵️‍♀️ Object Detection")
-endpoint =os.getenv("MODEL_ENDPOINT", default = "http://0.0.0.0:8000")
+endpoint =os.getenv("MODEL_ENDPOINT", default = "http://0.0.0.0:8000/detection")
```

Comment:
Model_Endpoint should only be host:port.
```suggestion
endpoint =os.getenv("MODEL_ENDPOINT", default = "http://0.0.0.0:8000")
```